selfdrive/updated.py

```diff
@@ -201,6 +201,11 @@ def finalize_update() -> None:
     shutil.rmtree(FINALIZED)
   shutil.copytree(OVERLAY_MERGED, FINALIZED, symlinks=True)
 
+  # Log git repo corruption
+  fsck = run(["git", "fsck", "--no-progress"], FINALIZED).rstrip()
```

Comment:
Won't git fsck throw a CalledProcessError if it returns with a non zero return code?

Reply:
I haven’t seen it exit with something non-zero, and the docs don’t specify any other return codes. We can keep an eye on this and see if it ever happens while it’s in master.